lib/auth/webauthncli/fido2_test.go

```diff
@@ -1816,6 +1910,13 @@ func (f *fakeFIDO2Device) Assertion(
 	pin string,
 	opts *libfido2.AssertionOpts,
 ) ([]*libfido2.Assertion, error) {
+	// Give preference to simulated errors.
+	if len(f.assertionErrors) > 0 {
```

Comment:
i would move it a little bit further in Assertion fn, for sure below initial validation.

Reply:
It could go either way - having it at the top allows us to fake validation errors as well, which could be useful. I considered placing it further down initially, but any place may or may not exactly match what the test wants.